src/main.c

```diff
@@ -46,11 +46,13 @@ static const int INS_NONE = -1;
 void ui_idle(void)
 {
 	currentInstruction = INS_NONE;
+
+	#if defined(TARGET_NANOS)
+	nanos_clear_timer();
+	h_expert_update();
 	// The first argument is the starting index within menu_main, and the last
```

Comment:
The first argument is nice when you wan to return from a submenu ti the menu element that triggered it.

Reply:
BTW you use the first argument below :).